ts/components/FiscalCodeLandscapeOverlay.tsx

```diff
@@ -46,13 +52,26 @@ const styles = StyleSheet.create({
 export default class FiscalCodeLandscapeOverlay extends React.PureComponent<
   Props
 > {
+  private scrollViewHeight: number = 0;
+  private ScrollVewRef = React.createRef<ScrollView>();
+
   private handleBackPress = () => {
     this.props.onCancel();
     return true;
   };
 
   public componentDidMount() {
     BackHandler.addEventListener("hardwareBackPress", this.handleBackPress);
+    if (this.props.showBackSide) {
+      setTimeout(() => {
+        if (this.ScrollVewRef.current) {
+          this.ScrollVewRef.current.scrollTo({
```

Comment:
```suggestion
          this.ScrollVewRef.current.scrollToEnd()
```
then you can remove scrollViewHeight